cmd/govim/main.go

```diff
@@ -312,9 +313,8 @@ func (g *govimplugin) Init(gg govim.Govim, errCh chan error) error {
 		g: g,
 	}
 
-	wd := g.ParseString(g.ChannelCall("getcwd", -1))
 	initParams := &protocol.ParamInitia{}
-	initParams.RootURI = string(span.FileURI(wd))
+	initParams.RootURI = string(span.FileURI(g.vimstate.workingDirectory))
```

Comment:
Agree, this is much less fragile for now. At least if it's wrong it will be consistently wrong from the point at which the user calls `cwd` onwards.

Reply:
👍